gbp/scripts/import_dsc.py

```diff
@@ -178,6 +178,8 @@ def apply_debian_patch(repo, source, dsc, upstream_commit, options):
         changes = get_changes(source.unpacked,
                               repo,
                               options.debian_branch)
+        # Commit messages are not allowed to contain NULs
+        changes = changes.replace('\0', '')
```

Comment:
Wouldn't it be better to that in `ChangeLog.get_changes ()` - this wold also avoid a component test and cold be tested in `tests/30_test_deb_changelog.py`.

Reply:
By the way, what’s the reason you don’t use python-debian to parse it?